tests/data/c_resources_per_pt/general/expected_summary.csv

```diff
@@ -1,18 +1,18 @@
-id,category,average,max
-ServiceRequest,* All,0.00,0
-Procedure,* All,0.00,0
-Observation,* All,0.00,0
-MedicationRequest,outpatient,1.33,2
-MedicationRequest,inpatient,0.33,1
-MedicationRequest,discharge; inpatient,0.33,1
-MedicationRequest,* No recognized category,0.33,1
-MedicationRequest,* All,2.33,4
-Immunization,* All,0.00,0
-Encounter,* All,0.00,0
-DocumentReference,* All,0.00,0
-DiagnosticReport,* All,0.00,0
-Device,* All,0.00,0
-Condition,* No recognized category,0.67,1
-Condition,* All,0.67,1
-AllergyIntolerance,* All,0.00,0
-* All,* All,3.00,4
+id,category,average,std_dev,max
+cumulus__all,cumulus__all,3.00,0.82,4
```

Comment:
I tested all these test values in an online standard deviation calculator and they all came out the same :shrug: